doc/reconstruction_models_list.rst

```diff
@@ -0,0 +1,232 @@
+.. raw:: html
+
+    <style>
+        table {
+        font-family: arial, sans-serif;
+        border-collapse: collapse;
+        width: 100%;
+        }
+
+        td, th {
+        border: 1px solid #dddddd;
+        text-align: left;
+        padding: 8px;
+        }
+
+        td.red { color: red;}
+        td.green {color: green;}
+        tr:nth-child(even) {
+        background-color: #f8f8f8;
+        }
+        </style>
+
+    <table>
+    <tr>
+        <th>Methods</th>
+        <th>Single Shell</th>
+        <th>Multi Shell</th>
+        <th>Cartesian</th>
+        <th>Ideal Data Requirements</th>
+        <th>Citations</th>
+    </tr>
+    <tr>
+        <td><a href='#diffusion-tensor-imaging'>DTI (SLS, WLS, NNLS)</a></td>
+        <td class='green'>Yes</td>
+        <td class='green'>Yes</td>
+        <td class='green'>Yes</td>
+        <td></td>
+        <td><a href='https://www.ncbi.nlm.nih.gov/pubmed/8130344'>Basser 1994</a></td>
+    </tr>
+    <tr>
+        <td><a href='#diffusion-tensor-imaging'>FwDTI</a></td>
+        <td class='red'>No</td>
+        <td class='green'>Yes</td>
+        <td class='red'>No</td>
+        <td></td>
+        <td><a href='https://www.ncbi.nlm.nih.gov/pubmed/19623619'>Pasternak 2009</a></td>
```

Comment:
Also - can you add here the reference for the official article of the the free water DTI implementation in dipy [[Henriques et al., 2017](https://github.com/ReScience-Archives/Henriques-Rokem-Garyfallidis-St-Jean-Peterson-Correia-2017/raw/master/article/Henriques-Rokem-Garyfallidis-St-Jean-Peterson-Correia-2017.pdf)]?

Henriques, R.N., Rokem, A., Garyfallidis, E., St-Jean, S., Peterson, E.T., and Correia, M.M. 2017. [Re] Optimization of a free water elimination two-compartment model for diffusion tensor imaging. ReScience 3, 1, #2. doi: 10.5281/zenodo.495237